noise:
  # X, Y, Z
  positionNoise: [0.0, 0.0, 0.0]

  # Yaw, Pitch, Roll
  orientationNoise: [0.0, 0.0, 0.001]  # EKF euler angle uses

  # X, Y, Z
  linearVelocityNoise: [0.1, 0.1, 0.0]  # imu velocity uses

  # X, Y, Z
  angularVelocityNoise: [0.0, 0.0, 0.1]  # imu velocity uses

  # X, Y, Z
  linearAccelerationNoise: [0.0, 0.0, 0.0]

  # lf, rf, lb, rb
  wheelSpeedNoise: [0.5, 0.5, 0.5, 0.5]  # wheel odom uses
